crtSignedContainer.sh

```diff
@@ -475,7 +475,7 @@ then
     test "$SB_PROJECT_INI_TRANS" && PROJECT_INI=$SB_PROJECT_INI_TRANS
 fi
 
-if [ "$PROJECT_INI" ]
+if [ "$PROJECT_INI" ] && [ "$SIGN_MODE" == "production" ]
```

Comment:
Please update the help text for the cli flag `--sign-project-config` to indicate that the value is ignored unless SIGN_MODE is production.